src/components/ChatEntry.js

```diff
@@ -1,22 +1,38 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
+  const heart = props.liked ? '❤️' : '🤍';
+  const handleLike = () => {
+    props.updateLike(props.id);
+  };
+  const senderClass =
+    props.sender === 'Vladimir' ? 'chat-entry local' : 'chat-entry remote';
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <div className={senderClass}>
```

Comment:
The only part of the class that changes based on being the local or remote sender is `local` or `remote`. `chat-entry` doesn't change. So consider only storing `local` or `remote` in `senderClass `, then modify this to
```js
    <div className={`chat-entry ${senderClass}`}>
```

This can help us avoid needing to build a huge combination of class name strings if we had other classes to toggle between.